src/stats/hist.jl

```diff
@@ -187,5 +190,8 @@ function Makie.plot!(plot::Hist)
         bp[1].val = points[]
         bp.width = w
     end
+    onany(plot.normalization, plot.scale_to, plot.weights) do _, _, _
+        notify(widths)
```

Comment:
```suggestion
        bp[1][] = points[]
```
This is simply more direct, there's no need to go through `widths` here.